beacon-chain/rpc/validator_server.go

```diff
@@ -98,6 +110,18 @@ func (vs *ValidatorServer) ValidatorCommitteeAtSlot(ctx context.Context, req *pb
 	if err != nil {
 		return nil, fmt.Errorf("could not fetch beacon state: %v", err)
 	}
+	if req.Slot%params.BeaconConfig().SlotsPerEpoch == 0 {
+		log.Info("Committee at slot is at epoch start")
```

Comment:
Can we minimize logging on RPC calls? Imagine we had 1000 clients connected then this would be flooding the logs.